gdx/src/com/badlogic/gdx/maps/tiled/BaseTmxMapLoader.java

```diff
@@ -351,6 +352,14 @@ protected void loadBasicLayerInfo (MapLayer layer, Element element) {
 		layer.setOffsetY(offsetY);
 		layer.setParallaxX(parallaxX);
 		layer.setParallaxY(parallaxY);
+
+		// tiled uses the format #AARRGGBB
+		// if the alpha of the tintcolor is set to 255, Tiled does not include it as part of the color code.
+		// ex. Red (r:255,g:0,b:0,a:255) becomes #ff0000, Red (r:255,g:0,b:0,a:127) becomes A:157 #7fff0000
+		String alpha = tintColor.length() == 9 ? tintColor.substring(1, 3) : "ff";
+		String color = tintColor.length() == 9 ? tintColor.substring(3) : tintColor.substring(1);
```

Comment:
Maybe a new static function for `Color`? Might be useful for others, and if we need it ourselves, it might be nice to make it available to others